Testing mapRotation property (#760)
Esri/arcgis-runtime-samples-java: map_view/map-rotation/README.md

## How to use the sample

> Press the A and D keys to rotate the map. If the map is not pointed north, the compass will display the current heading. Click the compass to set the map's heading to north.

Reviewer: ```suggestion
Press the A and D keys to rotate the map. The text will update to display the map's rotation value. If the map is not pointed north, the compass will display the current heading. Click the compass to set the map's heading to north.
```

## How it works

1. Create a `ArcGISMap` and set it to a `MapView`.

> 2. Create a `Compass` and pass the `MapView` to set the rotation angle of the map to the compass heading value.

Reviewer: let's break this up a little bit. I thought this sample's main point was to show the compass but I see it's primarily designed to show the map rotation functionality, so let's mention the compass as a last point. 

```suggestion
2. Use `mapView.mapRotationProperty().get()` to get the map's rotation angle.
3. Create a `new Compass(mapView)` to show the direction of north.
```

## Relevant API

> 

Reviewer: under `#Tags` let's add `toolkit` here, same as we do for the Scalebar sample. This would need a metadata update too

Reviewer: we should take viewpoint out here too following the sample changes

## How to use the sample

Press the A and D keys to rotate the map. The text will update to display the map's rotation value. If the map is not pointed north, the compass will display the current heading. Click the compass to set the map's heading to north.

## How it works

1. Create a `ArcGISMap` and set it to a `MapView`.

> 2. Use `mapView.mapRotationProperty().get()` to get the map's rotation angle.

Reviewer 2: ```suggestion
2. Use `mapView.getMapRotation()` to get the map's rotation angle.
```
We can use the getter here
